test/models/Applicant/Model.test.ts

```
@@ -109,6 +109,58 @@ describe("Applicant model", () => {
     });
   });
 
+  it("raise an error if name has a digit", async () => {
```

Aca tmb: `should raise an error if name has a digit`